invenio_rdm_records/records/api.py

```diff
@@ -99,6 +101,20 @@ class CommonFieldsMixin:
             pid_field=Affiliation.pid,
             cache_key='affiliations',
         ),
+        funding_funder=PIDListRelation(
+            'metadata.funding',
+            relation_field='funder',
+            attrs=['name'],
```

Comment:
when rebasing, this will be renamed to `keys`
```suggestion
            keys=['name'],
```

same below.